config/vufind/config.ini

```diff
@@ -58,15 +58,16 @@ theme           = bootprint3
 ; actual theme name.  For example, http://library.myuniversity.edu/vufind?ui=theme1
 ; would load the myTheme1 theme with the setting shown below.  Note that the values
 ; of "standard" and "mobile" are reserved for the default and mobile themes defined
-; above.
-;alternate_themes = theme1:myTheme1,theme2:myTheme2
+; above. Example: theme1:myTheme1,theme2:myTheme2
+alternate_themes = custom:custom_tmpl
```

Comment:
I thought this was a harmless way to see immediate results, but we can make it the commented-out example just the same.